Login[v2]: Improve login screens

mabartos opened the pull request:
- Closes #38120 
- Closes #38121

Related to: 

- https://github.com/keycloak/keycloak/issues/38117

### Before vs After:

<img width="390" src="https://github.com/user-attachments/assets/0748160c-1d7b-455c-a0cc-21ae36db701d"/>
<img width="390" src="https://github.com/user-attachments/assets/b6878b63-8eb7-49ae-a466-b6e154bb81b3"/>

### Before vs After:

<img width="390" src="https://github.com/user-attachments/assets/62547edb-f26a-41c7-864f-a665bcd01558"/>
<img width="390" src="https://github.com/user-attachments/assets/0f9c55a7-90d7-425b-ac0b-76d538d754f1"/>

### Before vs After:

<img width="390" src="https://github.com/user-attachments/assets/8fc1f94a-d737-41d9-9b58-b2bc0d74e998"/>
<img width="390" src="https://github.com/user-attachments/assets/341c19be-5242-4d5e-9255-2019a922f278"/>

------------------------------------

@jonkoops @edewit I believe the UI is nicer and more aligned with the v1 good layout.  Could you please check it?

mabartos committed:
Login[v2]: Improve login screens

Closes #38120

Closes #38121

Signed-off-by: Martin Bartoš <mabartos@redhat.com>

edewit commented on themes/src/main/resources/theme/keycloak.v2/login/resources/css/styles.css:
```diff
@@ -26,6 +37,10 @@ div.kc-logo-text {
     margin: 0 auto;
 }
 
+.kc-card {
+    border-top: 4px solid var(--keycloak-card-top-color);
+}
+
```
I must say I personally like this little detail, but don't we want to sick more to the way it was designed by PF and refrain from adding customization? https://v5-archive.patternfly.org/components/login-page/react/basic/

jonkoops replied:
If we are going to overwrite existing PatternFly conventions, let's not alias variables and use the original class (if possible). Any custom CSS that we add will make the eventual upgrade to a new PatternFly version more difficult, so we try to avoid it as much as possible.

If the intention here is to have a visual separator, there is a [Divider](https://v5-archive.patternfly.org/components/divider) component that we could use.

jonkoops replied:
Using the divider instead is not possible?

```html
<hr class="pf-v5-c-divider" />
```

jonkoops replied:
Ahhh, no, thanks for clarifying. I was under the impression that this was part of the separator between the form elements, consider this resolved.

jonkoops reviewed (commented):
I think this largely looks good, the visual appearance certainly is better. I have only one point that @edewit also raised that might have to be addressed.

mabartos committed:
Remove kc-card CSS property

Signed-off-by: Martin Bartoš <mabartos@redhat.com>

mabartos commented:
@jonkoops @edewit 

Rebased and removed the `kc-card` property. I set the border-top directly to the `pf-v5-c-login__main-header` as I didn't find a way using an utility classes for this one. 

I hope it's ok for now.
